### ISSUE TITLE
Infer code for user click

### ISSUE THREAD

**Issue by Robert0831** [feiaxyt/EMC-Click#7]
I want to infer image by user click point,i try write the code for infer, but the result is weird,I don't know where the mistake is.
Here is my code

import torch
from isegm.inference import utils
import cv2
import numpy as np

device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
path='hr18s.pth'
imgp='./train/images/000000000034.jpg'
model = utils.load_is_model(path, 'cuda')

image = cv2.imread(imgp)
image=cv2.resize(image,(640,400))
image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
image=torch.tensor(image).permute(2,0,1).unsqueeze(0)
prev_output = torch.zeros_like(image, dtype=torch.float32)[:, :1, :, :]
inp=torch.cat((image, prev_output), dim=1) 

point=[[[110,150,0],[300,300,0]]] # positive and negative point
point=torch.tensor(point)

eval_outputs, refined_output = model(inp.to(device), point.to(device), cached_outputs=None)

mask=torch.sigmoid(refined_output['instances_refined'])
mask=(mask.cpu().numpy()[:, 0, :, :] > 0.49).astype(np.int8)
mask=(mask[0]*255).astype(np.uint8)
cv2.imshow('Image', mask)
cv2.waitKey(0)
cv2.destroyAllWindows()

And it is my point
![dot](https://github.com/feiaxyt/EMC-Click/assets/121482640/c551b1b5-3e03-487f-a25e-0f2f3fd27bc2)
Here is the output
![output](https://github.com/feiaxyt/EMC-Click/assets/121482640/891af2af-b0d6-4a32-90ba-86887970e9e1)

Thanks!
